web-jehlomat/src/Components/Modal/Modal.tsx

```diff
@@ -1,18 +1,18 @@
-import { FC, useState } from 'react';
+import { FC } from 'react';
 import { useMediaQuery } from '@mui/material';
 import styled from '@emotion/styled';
-import { white, primaryDark } from '../../utils/colors';
+import { white } from '../../utils/colors';
 import { media } from '../../utils/media';
-import AddButton from '../Buttons/AddButton/AddButton';
-import TextButton from '../Buttons/TextButton/TextButton';
-import { Modal } from '@mui/material';
+import MuiModal, { ModalProps } from '@mui/material/Modal';
 import TitleBar from '../../Components/Navigation/TitleBar';
 import ChevronLeft from '@mui/icons-material/ChevronLeft';
 import CloseIcon from '@mui/icons-material/Close';
 
-interface Props {
-    buttonText: string;
+type CloseFunction = () => void;
```

Comment:
Asi není důležité to takto typovat, klidně bych to dal rovnou do té onClose. 

tzn.

```
onClose: () => void
```

Reply:
to pak řve eslint :)

Reply:
To je zvláštní... Nic takového mi to neříká. `npm run lint` ti prochází (když si předtím pustíš `npm run fix`)